CHANGELOG.md

```diff
@@ -382,13 +384,59 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
+### Migration steps
+
+- Due to a bug in the PUT `/rules/<name>` endpoint, the rule records in PostgreSQL may be
+out of sync with records in DynamoDB. In order to bring the records into sync, re-run the
+[previously deployed `data-migration1` Lambda](https://nasa.github.io/cumulus/docs/upgrade-notes/upgrade-rds#3-deploy-and-run-data-migration1) with a payload of
+`{"forceRulesMigration": true}`:
+
+```shell
+aws lambda invoke --function-name $PREFIX-data-migration1 \
+  --payload $(echo '{"forceRulesMigration": true}' | base64) $OUTFILE
+```
+
+### Added
+
+- **CUMULUS-2846**
```

Comment:
I think I would expect to see these CHANGELOG lines from `master` here?

https://github.com/nasa/cumulus/blame/master/CHANGELOG.md#L29-L32

Reply:
Nevermind. It seems like the reason why is that these changes are from `master` before 2845 was merged into it. And we shouldn't add the latest from `master` into this PR, we'll just do another PR subsequently.